docs/configuration.md

```diff
@@ -326,4 +326,209 @@ deployment_config = DeploymentConfig(
        'docker_repository': 'your_repository',
        'gcp_project_id': '{project_id}',
        'dags_bucket': '{dags_bucket}'}})
-```
\ No newline at end of file
+```
+
+
+# Class-based configuration [experimental]
+
+Bigflow 1.3 introduces new class-based approach for keeping configurations.
+New API is optional and there is no need to migrate existing code from
+`biglfow.Configuration`. However it betters plays with typehinting, enables
```

Comment:
However**,** it

Reply:
However it betters plays with typehinting -> However, it allows you to use type hints, which enables...